src/js/components/Video/stories/typescript/ControlsBelow.tsx

```diff
@@ -4,7 +4,7 @@ import { Box, Video } from 'grommet';
 
 const SimpleVideo = (props) => (
   <Box align="center" pad="large">
-    <Video {...props}>
+    <Video skipInterval={2} {...props}>
```

Comment:
I am having troubles accessing the player controls after I play the video. They are available on first render, but if I play, they get hidden and I am no longer able to access them with the keyboard. Am I doing something wrong?

Reply:
resolve in latest commit